packages/data/lib/services/analytics_service_impl.dart

```diff
@@ -9,7 +9,10 @@ class AnalyticsServiceImpl implements AnalyticsService {
   AnalyticsServiceImpl(this._analyticsProvider);
 
   @override
-  Future<void> logEvent(String eventName) async {
-    await _analyticsProvider.logEvent(name: eventName);
+  Future<void> logEvent(String eventName, [String? payload]) async {
```

Comment:
pass payload as map, not as string, since it can contain multiple params

Reply:
Fixed.